script/Arcueid/Day 8/08_00_ARC08_3_1.txt

```diff
@@ -331,57 +372,64 @@ In fact, there aren't any incidents or trouble happening in school, and there's
 [sha:c23335af925751e582dfc8002b18ac5fecfd8832]{
 -- Page 186, Offset 20129.
 -- 志貴がそう言うんなら、ここに異常はないかもしれない」
-If that's what you say, then there's probably nothing wrong here."
+I'll take your word for it,
+ there's probably nothing out of place here."
 }
 [sha:9c45ab950da0eae8346c9c65af751e6368de895d]{
 -- Page 186, Offset 20130.
 -- 「ないかもしれない、じゃなくてないんだって、実際」
-"It's not probably, there's definitely nothing wrong here."
+"It's not %{i}probably%{/i}, there's %{i}definitely%{/i} nothing wrong here."
 }
 [sha:69031f8c51b7cba6bf4f211dc583b93686984ba1]{
 -- Page 186, Offset 20131.
 -- アルクェイドは納得のいかない顔をしたままだ。
-An unconvinced look remains on Arcueid's face.
+The look on Arcueid's face makes it clear she's still not convinced.
 }
 [sha:94bc8a707f72721b43e676420201ad05df4f9117]{
 -- Page 187, Offset 20132.
 -- 「―――やば、昼休みになっちゃったか」
-"―――Crap, it's already lunch break."
+"―――Crap, it's already lunch."
 }
 [sha:e5e7ee86562fc8f12438b1c94dc33fbc28a32395]{
 -- Page 187, Offset 20133.
 -- いくら体育館裏に人気がないといっても、昼休みになれば通りかかる生徒が出てくるだろう。
-Even if the back of the gym remains rather unpopulated, there will still be students passing by now that it's lunch.
+While the back of the gym isn't exactly a popular destination,
+ there will still be one or two students passing by on their lunch break.
 }
 [sha:1f587ba8b7ddf7950aba09e7f18551c21913e2ba]{
 -- Page 188, Offset 20134.
 -- 「とにかく、これ以上ここにいられると騒ぎになる。
-"Anyway, it will become a riot if you stay here any longer.
+"Anyway, you're going to cause a commotion if you stick around any longer.
 }
 [sha:53381efc193be281c9f75afc7aa04751ab9d4a7b]{
 -- Page 188, Offset 20135.
 -- ちゃんと約束は守るから、おまえは早く帰って体を休めておくこと。今日の夜にも“敵”とやらに出会うかもしれないんだから」
-I swear I won't break our promise, so just go home and get some rest already. We might run into an enemy or something tonight after all."
+I swear I won't break our promise, so just go home and get some rest already.
+ We might run into more enemies tonight, you never know."
 }
 [sha:ea0a014c64648859f6e7e2d38774de217da05175]{
 -- Page 189, Offset 20136.
 -- 「……ふーん。なんだかわたしを追い出したいみたいな口ぶりね、志貴。そんなに<学|こ><校|こ>にいられると迷惑？」
-"...Hmm. It sounds awfully like you're trying to get rid of me. Does it really bother you that much if I stay here?"
+"...Hmm. It sounds an awful lot like you're trying to get rid of me.
+ Is my presence here really that much of a nuisance for you?"
 }
 [sha:184eeaf260fbb18cda78aef09af407ca610fa635]{
 -- Page 190, Offset 20137.
 -- 「迷惑―――じゃないけど、いろいろ頭が痛いんだよ！
-"It's not that it bothers me, but that my head hurts from a lot of things!
+// Assuming this is figurative
+"It's not that you're a nuisance,
+ I just have a lot of things to deal with right now!
```

Comment:
"headache" works as troubles in english too. maybe something like "Life is just full of headaches right now"?